python/lsst/pipe/analysis/plotUtils.py

```diff
@@ -924,3 +924,39 @@ def determineExternalCalLabel(repoInfo, patch, coaddName="deep"):
             uberCalLabel += "SFM"
 
     return uberCalLabel
+
+
+def plotRhoStats(axes, rhoStats):
+    """Plot all five Rho Statistics
+
+    Parameters
+    ----------
+    axes : `list`
+        A list containing two `matplotlib.figure.ax` handles
+    rhoStats : `dict`
```

Comment:
I've just learnt this from being reviewed: the dict needs more details. https://developer.lsst.io/python/numpydoc.html#dictionary-types